File: app/src/main/java/com/example/mobile/SplashActivity.java
package com.example.mobile;

import android.content.Intent;
import android.os.Bundle;
import android.os.*;

import androidx.appcompat.app.AppCompatActivity;

public class SplashActivity extends AppCompatActivity {
        @Override
        protected  void onCreate(Bundle savedInstanceState){
            super.onCreate(savedInstanceState);

            new Handler().postDelayed(new Runnable(){
                @Override
                public void run() {
                    Intent intent = new Intent(SplashActivity.this, LoginActivity.class);
                    startActivity(intent);
                    finish();
                }
            }, 700);
        }
}

File: app/src/main/java/com/example/mobile/SplashActivity.kt
package com.example.mobile

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import androidx.appcompat.app.AppCompatActivity

class SplashActivity : AppCompatActivity() {
    private val splashTime:Long = 1500
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_logo)

        Handler().postDelayed({
            startActivity(Intent(this, LoginActivity::class.java))
            finish()
        },splashTime)

    }
}